app/components/collections/Thumbnails.tsx

```diff
@@ -59,22 +108,28 @@ const Thumbnails = ({ collectionType, children }: Props) => {
                   </figcaption>
                 </figure>
               </li>
-            );
-          })}
-        </ol>
-        <Pagination
-          classNames={{
-            root: "justify-self-center px-2 py-4 bg-white w-full md:w-2/3 lg:w-2/5",
-            list: "flex flex-row items-stretch justify-center",
-            pageItem:
-              "bg-county/20 text-white mx-4 text-center rounded-md min-w-6 max-w-8",
-            selectedItem: "bg-county text-white",
-          }}
-          padding={2}
-        />
-      </div>
+            ))}
+          </ol>
+          <Pagination
+            classNames={{
+              root: "justify-self-center px-2 py-4 bg-white w-full md:w-2/3 lg:w-2/5",
+              list: "flex flex-row items-stretch justify-center",
+              pageItem:
+                "bg-county/20 text-white mx-4 text-center rounded-md min-w-6 max-w-8",
+              selectedItem: "bg-county text-white",
+            }}
+            padding={2}
+          />
+        </>
+      ) : (
+        <div className="h-[700px] mt-6 mx-2 rounded-md overflow-hidden">
+          <ClientOnly>
+            {() => <Map className="w-full h-full" geojson={geojson} />}
```

Comment:
The Map component does not take geojson. You can look [here](https://github.com/ecds/georgia-coast-atlas-frontend/blob/dev/app/components/mapping/PlaceGeoJSON.tsx) as an example. That's not great because the RelatedPlacesMap component expects to be within the context of a place. You could repurpose the [PlaceGeoJSON](https://github.com/ecds/georgia-coast-atlas-frontend/blob/dev/app/components/mapping/PlaceGeoJSON.tsx) component. I actually removed that from my branch for the unified UI. It will not work for you as is, but you can rework it as it will not be needed in the near future.